src/foam/util/DeFeedback.js

```diff
@@ -0,0 +1,29 @@
+/**
+* @license
+* Copyright 2022 The FOAM Authors. All Rights Reserved.
+* http://www.apache.org/licenses/LICENSE-2.0
+*/
+
+foam.CLASS({
+  package: 'foam.util',
+  name: 'DeFeedback',
```

Comment:
Maybe this would be a good feature for listeners in general? feedback: false

If we turn out using this fairly often, we might want to consider that.